### ISSUE TITLE
I am not able to track changes made on resx files in vscode's source control

### ISSUE THREAD

**Issue by magmukendi** [pmahend1/resxpress#79]
I am currently using: 
- vscode 1.85.1
- reXpress v5.7.0

I am trying to track the changes I have made on Visual Studio code's source control, but the changes made don't appear on the resx file as they usually appear on another file such as source code, readme file, or xml etc.

In the following screenshot as an illustration, I have changed the value in the first column from `Add migration`  to `Remove migration` but this is not reflected in the resx when open from the source control for tracking changes.

![image](https://github.com/pmahend1/resxpress/assets/64711820/12cc74d2-553d-434f-aae1-7b7383c895ca)

Usually when the changes made are displayed, the deleted characters are highlighted in red and the added characters are highlighted in green. I have provided a screenshot below for illustration.

![image](https://github.com/pmahend1/resxpress/assets/64711820/5d574b6c-28c8-49a7-a905-469077bc4cd6)

I am not sure if this issue should be reported to vscode or the resxpress extension. It would be great if you could point me to the right direction to get this issue fixed for other developers that would go through the same issue.

Happy New Year, Cheers 🥂!

**Comment by pmahend1** [comment]
Thanks for reporting. I will take a look. My understanding is that git compare uses the default editor set to display the differences which I am not sure can be changed. Are you using an extension like GitLens-Supercharged?

**Comment by magmukendi** [comment]
Yes, I do have GitLens supercharged up and running. 

Thanks for checking out. Just let me know if this is something that can be improved on your side or if I need to reach out to the vscode team directly.

**Comment by pmahend1** [comment]
I am not sure. I will check. It may have to do with GitLens may be as that seems to be calling vscode default editor for checking changes.

**Comment by pmahend1** [comment]
Here are the commands which are triggered when we try to see file differences.  You may try configuring **git difftool** and see if it lets you see only text differences. Otherwise as a work-around you can temporarily disable extension while committing. I will see check further if there is alternative, so far I haven't found one.

```shell
git show --textconv file1.resx
git ls-files --stage -- file1_modified.resx
git cat-file -s xxxxxxxxxxxxxxxxxx
```

**Comment by pmahend1** [comment]
@magmukendi An alternative solution is for me to [make `Resx Editor` as an option and not default](https://code.visualstudio.com/api/extension-guides/custom-editors#contribution-point)

> - priority - (optional) Specifies when the custom editor is used.
> priority controls when a custom editor is used when a resource is open. Possible values are:
>     - "default" - Try to use the custom editor for every file that matches the custom editor's selector. If there are multiple >custom editors for a given file, the user will have to select which custom editor they want to use.
>   - "option" - Do not use the custom editor by default but allow users to switch to it or configure it as their default.

Downside of this is everytime you want to open resx you will need to right click and choose open with Resx Editor.

**Comment by pmahend1** [comment]
Fixed by #81
